.github/workflows/ci-cd.yml

```diff
@@ -421,9 +420,15 @@ jobs:
       uses: pypa/cibuildwheel@v2.22.0
       env:
         CIBW_ARCHS_MACOS: x86_64 arm64 universal2
-    - uses: actions/upload-artifact@v3
+    - if: ${{ matrix.qemu }}
+      uses: actions/upload-artifact@v4
       with:
-        name: dist
+        name: dist-${{ matrix.os }}-${{ matrix.qemu }}
```

Comment:
The last bit could just be made conditional instead of adding another step:
```suggestion
        name: >-
          dist-${{ matrix.os }}-${{
            matrix.qemu
            && matrix.qemu
            || 'native'
          }}
```

Reply:
In other places, we don't really care about the conditional even: https://github.com/aio-libs/multidict/blob/fbf28ae/.github/workflows/reusable-build-wheel.yml#L79

Reply:
It the duplicate a mistake @webknjaz? `matrix.qemu && matrix.qemu`

Reply:
It works perfectly as suggested, without it is evaluated as a boolean expression.